src/cmd/tools/dtest/docker/harness/resources/harness.go

```diff
@@ -259,3 +325,17 @@ func (r *dockerResources) Cleanup() error {
 
 func (r *dockerResources) Nodes() Nodes             { return r.nodes }
 func (r *dockerResources) Coordinator() Coordinator { return r.coordinator }
+
+func attachToExistingDBNode(
+	dbNodes Nodes,
+	pool *dockertest.Pool,
+	containerName string,
+	iOpts instrument.Options,
+) (Nodes, error) {
+	dbNode, err := attachToExistingNode(pool, containerName, iOpts)
+	if err != nil {
+		return dbNodes, err
+	}
+	dbNodes = append(dbNodes, dbNode)
+	return dbNodes, nil
+}
```

Comment:
This seems to be used only in a single place. I think inlining it would make the code slightly clearer.